corehq/apps/hqwebapp/forms.py

```diff
@@ -99,6 +93,33 @@ def __init__(self, plural_noun, action, form_id, *args, **kwargs):
             ),
         )
 
+    def crispy_form_fields(self, context):
+        return [
+            crispy.Field(
+                'action',
+            ),
+            crispy.Field(
+                'bulk_upload_file',
+                data_bind="value: file",
+            )
+        ]
+
+
+class AppTranslationsBulkUploadForm(BulkUploadForm):
```

Comment:
@orangejenny do you have suggestion on doing this better?
i had to add two fields to BulkUploadForm which is used at multiple places. My first approach was [this](https://github.com/dimagi/commcare-hq/pull/20939/commits/ef459ccfe7748bc406775a1425dd860f779390cc#diff-d39e56548bf4e0c07edcb0998f4aef34). But then i decided to make it less complicated by adding a new class instead of modifying `BulkUploadForm`.

Reply:
This seems all right; I like subclassing better than the previous code that added logic to `BulkUploadForm`. This class should into corehq/apps/app_manager/forms.py

Reply:
haha.. I found editing `BulkUploadForm` really ugly @esoergel .. by adding this subclass it helps extract out the logic required only for `AppTranslationsBulkUploadForm` separately so don't think one editing `BulkUploadForm` would break anything here.